templates/user_order.html

```diff
@@ -57,4 +58,4 @@ <h5 class="card-title">User Order</h5>
             });
         });
     </script>
-{% endblock %}
```

Comment:
eof change

Reply:
should now have no newline at end of file